lib/datastore/transaction.js

```diff
@@ -85,16 +85,18 @@ nodeutil.inherits(Transaction, DatastoreRequest);
  *     a transaction.
  *
  * @example
- * transaction.begin(function(err) {
- *   // Perform Datastore operations as usual.
- *   transaction.get(dataset.key(['Company', 123]), function(err, entity) {
- *     // Commit the transaction.
- *     transaction.finalize(function(err, apiResponse) {});
+ * dataset.runInTransaction(function(transaction, done) {
```

Comment:
All of the examples in this file have been wrapped with `dataset.runInTransaction()` however the code inside the callback is not tested because the function is async so for example in this one, `transaction.begin`, `transaction.get` and `transaction.finalize` are never tested (run), except for syntax when the code block is compiled (so it's not a total loss).